.github/workflows/unit-test.yml

```diff
@@ -53,8 +53,10 @@ jobs:
           USE_GPU: "False"
         run: |
           python -m unittest discover tests &&
+          echo "Checking that optional dependencies are not needed" &&
```

Comment:
is it possible to start from a clean environment? otherwise we need to update this test for each new extra dependency

Reply:
It's quite difficult, as the image used for unit tests already contains all the required dependencies. Una method is to parse [extra_dependencies.txt](https://github.com/ContinualAI/avalanche/blob/master/extra_dependencies.txt), but this would require an ad-hoc script to do it.

Reply:
is it possible to use a different image?

Reply:
Yes, but that would also require changing the docker image creation actions. It may be a bit overkill.